src/service/standard-actions-impl.js

```diff
@@ -163,6 +166,17 @@ export class StandardActions {
   }
 
   /**
+   * @param {!./action-impl.ActionInvocation} invocation
+   * @private
+   */
+  handleAmpPrint_(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return;
+    }
+    self.print();
```

Comment:
instead of `self`, let's use `const win = (node.ownerDocument || node).defaultView;` , similar to `handleAmpNavigateTo_`